webapp/src/widget/menuItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import CompassIcon from '../components/icons/compassIcons';
+
+type Props = {
+    action: () => void,
+    icon: string,
+    text?: string,
+    shortcut?: string,
+}
+
+const MenuItem = (props: Props) => {
+    const { icon, shortcut, action, text } = props;
+
+    useEffect(() => {
+        function handleKeypress(e: {key: string}) {
+            if (e.key === shortcut) {
```

Comment:
If shortcut changes during the lifetime of the component, this won't change. Is this expected?